server/controllers/FollowController.js

```diff
@@ -0,0 +1,202 @@
+import db from '../database/models';
+
+const { User, Follow } = db;
+
+/**
+ * Class representing FollowController
+ */
+class FollowController {
+  /**
+   * Handles the Follow Feature
+   * @param {*} req - Request object
```

Comment:
Maybe you can try and update all the comments to use something similar to this ``@param {object} req - Request object `` instead of ``@param {*} req - Request object``. Every other thing looks good to me. Great job

Reply:
okay..